src/operator/utils.ts

```diff
@@ -8,6 +8,11 @@ import {
 // Types
 import {OperatorOrgLimits, OperatorRegions} from 'src/types'
 
+type OperatorRole = 'read-only' | 'read-write' | null
+
+export const isUserOperator = (operatorRole: OperatorRole): boolean =>
+  operatorRole === 'read-only' || operatorRole === 'read-write'
+
 const updateMaxRetentionWithCallback = (
```

Comment:
Ended up breaking this out into a separate function - there's a place this will also be useful in `TreeNav`.